.github/tests-visual/visual-record.js

```diff
@@ -240,7 +245,12 @@ async function main() {
 
             // run through all pages
             for (var fileName in allPages) {
-                await page.goto(allPages[fileName], { waitUntil: 'load' });
+                const url = allPages[fileName]
+                const response = await page.goto(url, { waitUntil: 'load' });
+                if (!response.ok()) {
```

Comment:
Kann man dieses Abfangen noch irgendwie verallgemeinern, sodass es auch bei den Screenshots greift, die wir außerhalb der for-schleife separat machen?

Reply:
ja können wir, würde ich separaten PR machen, um diesen nicht weiter zu vergrößern.

Reply:
hab ich gemacht

Reply:
oder markus macht das schon :wink: